pylint/message/message_definition_store.py

```diff
@@ -32,6 +33,28 @@ def __init__(self) -> None:
         # MessageDefinition kept by category
         self._msgs_by_category: dict[str, list[str]] = collections.defaultdict(list)
 
+    @staticmethod
+    def is_deleted_msgid_or_symbol(msgid_or_symbol: str) -> str | None:
+        """Return the explanation for removal if the message was removed."""
+        for explanation, deleted_messages in DELETED_MESSAGES.items():
+            for deleted_message in deleted_messages:
+                if msgid_or_symbol in [deleted_message.msgid, deleted_message.symbol]:
+                    return explanation
+        return None
+
+    def kwargs_for_bad_option_value_add_message(
```

Comment:
Let's make both of these private while we haven't documented our public API specifically.

Reply:
We're using it in PyLinter ourself so it's complicated to make them private. I'm also ok with poeple using the ``is_deleted_msgid_or_symbol`` as a public API. What about making ``kwargs_for_bad_option_value_add_message`` protected ?